content/guides/remote-development.md

```diff
@@ -0,0 +1,353 @@
+
+ * A ContinuousPipe hosted project with the GitHub integration set up
+ * The project checked out locally
+ * The IP address, username and password to use for Kubenetes cluster
+ * rsync installed locally
+ * A [keen.io](https://keen.io) write token, project id and event collection name if you want to log usage stats
+
+Note: if the GitHub repository is not the origin of your checked out project then you will
+need to add a remote for that repository.
+
+## Installation
+
+OSX (64bit):
+
+```
+sudo curl https://continuouspipe.github.io/remote-environment-client/0.0.1/darwin-amd64.gz > cp-remote.gz
+gzip -d cp-remote.gz;
+mv cp-remote /usr/local/bin/cp-remote
+chmod +x /usr/local/bin/cp-remote
+```
+
+Dependencies: You need to have 'git', and 'rsync' installed and available in the shell where cp-remote runs
+
+Linux (64-bits):
+
+if your architecture is 32bit use linux-386.gz rather than linux-amd64.gz
+
+```
+sudo curl https://continuouspipe.github.io/remote-environment-client/0.0.1/linux-amd64.gz > cp-remote.gz
+gzip -d cp-remote.gz;
+mv cp-remote /usr/local/bin/cp-remote
+chmod +x /usr/local/bin/cp-remote
+```
+
+Dependencies: You need to have 'git', and 'rsync' installed and available in the shell where cp-remote runs
+
+Windows (64-bits):
+
+if your architecture is 32bit use windows-386.gz rather than windows-amd64.gz
+
+* Download https://continuouspipe.github.io/remote-environment-client/0.0.1/windows-amd64.gz
+* Extract cp-remote.gz
+* Move cp-remote.exe into your project folder
+
+Dependencies: You need to have 'git', and 'cwRsync' installed and available in your environment PATHS variable
+
+### Migrate to CP-Remote Go
+
+- Install the latest version, see instructions in the Installation section above
+- In your project directory, run `cp-remote setup` (see Setup section below)
+
+**Changes in command arguments**
+
+In the Go version, the commands arguments needs to be passed as flags.
+To find out information about the available flags for each command run `cp-remote [command] --help` or `cp-remote [command] -h`
+
+**Examples:**
+
+Previously to open a bash remote shell overriding the default service,
+the command using the bash script would have been `cp-remote bash web`.
+This has been updated to `cp-remote bash -s web`, alternatively you may use the full flag name `cp-remote bash --service web`
+
+To execute a command onan environment which differs from the default one.
+The previous command would have been `cp-remote exec --namespace=project-key-feature-my-shiny-new-work -- ls -l`.
+This has been updated to `cp-remote exec --project-key example --remote-branch feature-my-shiny-new-work -- ls -l`.
+Alternatively a more concise version is `cp-remote exec -p example -r feature-my-shiny-new-work -- ls -l`.
+
+## Setup
+
+```
+cp-remote setup
+```
+
+To start using this tool for a project, run the `setup` command from the project root.
+ This will install kubectl if you do not have it installed already. It will then
```

Comment:
`This will install kubectl if you do not have it installed already` Doesn't the go version get bundled with `kubectl`. I don't think its available independently to the user after installing `cp-remote`